src/app.js

```diff
@@ -46,7 +46,8 @@ router.post('/courses', coursesController.addCourse);
 // router.post('/courses/{courseId}/guides/{guideId}/exercises', coursesController.getCourses);
```

Comment:
remember that you should not add any route with `/exercises`

Reply:
also, check this route because I am not sure if we should add it (bah, I don't know) `router.patch('/courses/{courseId}/users/{userId}', coursesController.getCourses);`

Reply:
I'm going to use that endpoint to update the users role in the course